README.md

```diff
@@ -213,6 +214,8 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Rust-Clippy](https://github.com/rust-lang-nursery/rust-clippy/labels/good%20first%20issue) _(label: good first issue)_  <br> A bunch of lints to catch common mistakes and improve Rust code
 - [Rustfmt](https://github.com/rust-lang-nursery/rustfmt/labels/good-first-issue) _(label: good-first-issue)_ <br> A tool for formatting Rust code according to style guidelines.
 - [Iron](https://github.com/iron/iron/labels/easy) _(label: easy)_ <br> An extensible, concurrent web framework for Rust
+- [TiKV](https://github.com/tikv/tikv/issues?q=is%3Aissue+is%3Aopen+label%3A%22D%3A+Easy%22) _(label: for new contributors, Difficulty: Easy. Ideal for beginners, with mentoring available!)_ <br> A distributed transactional key-value database
```

Comment:
can you please use the same format for links as around? directly to labeled issues, not filter

Reply:
Changed. Please take a look. Thanks!